src/app/tests/suites/certification/Test_TC_BI_1_1.yaml

```diff
@@ -35,8 +37,160 @@ tests:
+
+    - label: "Reads back mandatory global attribute: CommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "CommandList"
+      response:
+          value: 0
+
+    - label: "Read the global attribute: EventList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+
+    - label: "Write the default values to mandatory global attribute: EventList"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "EventList"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back mandatory global attribute: EventList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          value: 0
+
+    - label: "Read the global attribute: ClientGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ClientGeneratedCommandList"
+      response:
+          constraints:
+              type: list
+
+    - label:
+          "Write the default values to mandatory global attribute:
+          ClientGeneratedCommandList"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "ClientGeneratedCommandList"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back mandatory global attribute: ClientGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ClientGeneratedCommandList"
+      response:
+          value: 0
+
+    - label: "Read the global attribute: ServerGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ServerGeneratedCommandList"
+      response:
+          constraints:
+              type: list
+
+    - label:
+          "Write the default values to mandatory global attribute:
+          ServerGeneratedCommandList"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "ServerGeneratedCommandList"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back mandatory global attribute: ServerGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ServerGeneratedCommandList"
+      response:
+          value: 0
+
+    - label: "Read the optional global attribute : FeatureMap"
+      disabled: true
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: map32
+
+    - label: "Write the default values to optional global attribute: FeatureMap"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "FeatureMap"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back optional global attribute: FeatureMap"
+      disabled: true
+      command: "readAttribute"
+      attribute: "FeatureMap"
```

Comment:
The question is and maybe i m not aware of this !
Does a global attribute featureMap is writable or Read Only ?

Reply:
As per the Spec its is Read only
![image](https://user-images.githubusercontent.com/84442963/143418434-8fa32539-926b-4ab9-9613-97334621a3c9.png)

Reply:
OK ! so we have a problem there 
You cannot expect the feature Map to be ZERO for ANY Cluster this is wrong
You gonna make every single cluster fails at this test

Reply:
But as per Testplan we need to Read , Write and Read back the default value and we know that feature map is read only for that we are passing error message as 1 it will give us that attribute is not writable 
Testplan screenshot:
![image](https://user-images.githubusercontent.com/84442963/143809872-d3b5e433-363e-4b67-ba16-aacc692ea53c.png)
![image](https://user-images.githubusercontent.com/84442963/143809803-1f021ce5-493c-47fe-8c3e-92f2046c6ac4.png)

Reply:
Well there are likely 2 mistakes here 
- 1 one in the spec about the default value zeroed
- 2 in the expression of the test plan about the default 

  FeatureMap exits to express the capabilities of a device then why a device will ever expose a featuremap if it has no extra capabilities to express . ? that make no sense ; right ? You can look at some endpoint config feature map is not zeroed most of the time your PR wont pass the CI

Reply:
we will raise spec and test plan issues and add the ticker number